tensorflow/python/kernel_tests/signal/fft_ops_test.py

```diff
@@ -158,6 +158,31 @@ def _tfIFFTForRank(self, rank):
     else:
       raise ValueError("invalid rank")
 
+  #test cases for fftshift and ifftshift
```

Comment:
Instead of this comment, please make a new TestCase class for these:
```python
class FFTShiftTest(test.TestCase):
   ...
```

Reply:
done. 👍